lookup.js

```diff
@@ -289,6 +305,27 @@ exports.init = function (sbot, config) {
     )
   }
 
+  function isRootFeedId(feedId, cb) {
+    try {
+      assertFeedId(feedId)
+      if (!validate.detectFeedFormat(feedId)) throw Error('Invalid feedId')
+    } catch (err) {
+      return cb(err)
+    }
+
+    sbot.db.query(
+      where(parent(feedId)),
```

Comment:
I'm not sure this is the appropriate indicator of being a root metafeed. In `query.js` (by the way, why is this PR for lookup.js and not query.js?) we use `getAnnounces()` to discover root MFs, and I think that's a more sure way. Because it's not hard for a developer to randomly use `'v1'` as a feedpurpose somewhere for their own leaf, but then suddenly this function would think that e.g. a shard is a root.

Reply:
@mixmix

Reply:
I put it in `lookup.js` because it felt like the analogue to the lookup in that was happening in the other direction. Happy to move it if there's somewhere more sensible.

Will look into your suggestion about `getAnnounces` and come back

Reply:
have moved to use @staltz 's suggestion, and put it in `query.js`

This will need to change if we ever decide to drop the `main` feed, but can figure that out later